backend/analytics_server/mhq/service/code/pr_analytics.py

```
@@ -1,6 +1,8 @@
+from mhq.store.models.code.filter import PRFilter
+from mhq.store.models.core.teams import Team
```

Flake8 (7.3.0)

[error] 'mhq.store.models.core.teams.Team' imported but unused

(F401)